### ISSUE TITLE
Translation to Polish

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ejwa/gitinspector#16]
```
New translation for Polish language. .po files in attachment
```

Original issue reported on code.google.com by `kamila.chyla` on 23 Aug 2013 at 8:44

Attachments:
* [messages_pl.po](https://storage.googleapis.com/google-code-attachments/gitinspector/issue-16/comment-0/messages_pl.po)

**Comment by adam-waldenberg** [comment]
Hi. A new messages.pot has been added with commit 47b11ad, so an update is needed to the translation file. Refer to the commit diff to see all changes.

Once updated, the translation can be included in the upcoming 0.4.2 release.

**Comment by adam-waldenberg** [comment]
Closing. Update posted with pull request #79.
